src/Synapse/Synapse/Models/SynapseAnalyticsArtifactsClient.cs

```diff
@@ -88,9 +89,11 @@ public void DeletePipeline(string pipelineName)
             _pipelineClient.StartDeletePipeline(pipelineName);
         }
 
-        public CreateRunResponse CreatePipelineRun(string pipelineName, string referencePipelineRunId, bool? isRecovery, string startActivityName, IDictionary<string, object> parameters)
+        public string CreatePipelineRun(string pipelineName, string referencePipelineRunId, bool? isRecovery, string startActivityName, IDictionary<string, object> parameters)
```

Comment:
Changing the return type to string would affect `InvokeAzureSynapsePipeline.cs`, which was designed to return a `PSCreateRunResponse`

Reply:
It's a bug need to fix. In the next released package of Artifacts, the return type will be CreateRunResponse.